docs/glossary.md

```diff
@@ -0,0 +1,62 @@
+# OLM Glossary
+
+A list of OLM terms, with definitions and common acronyms.
+
+## CustomResourceDefinitions
+
+### CatalogSources
+
+**Definition**: The CatalogSource is a repository of [CSVs](#ClusterServiceVersion), [CRDs](#CustomResourceDefinitions), and packages that define an application. 
+**Acronyms**: CatSrc(s)
+
+### ClusterServiceVersion
+
+**Definition**: The ClusterServiceVersion is a required resource for an Operator which includes application metadata such as name, description, version, repository link, labels, icon, etc. It also includes a list of corresponding/required resources and their statuses, install strategy of the Operator as [deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/), supported install modes, etc.
+**Acronyms**: CSV(s)
+
+### InstallPlans
+
+**Definition**: The InstallPlan lists out all resources to be created in order to automatically install or upgrade a [CSV](#ClusterServiceVersion). 
```

Comment:
Similarly for InstallPlan definition: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/philosophy.md#installplan
InstallPlan can have a install strategy of either manual or automatic, not sure the use of the word `automatically` is the best move here.

Reply:
nice catch